packages/config/src/config/database.test.ts

```diff
@@ -0,0 +1,50 @@
+/*
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/.
+ *
+ * OpenCRVS is also distributed under the terms of the Civil Registration
+ * & Healthcare Disclaimer located at http://opencrvs.org/license.
+ *
+ * Copyright (C) The OpenCRVS Authors. OpenCRVS and the OpenCRVS
+ * graphic logo are (registered/a) trademark(s) of Plan International.
+ */
+import * as mongoose from 'mongoose'
+import { start, stop } from '@config/config/database'
+import { logger } from '@config/config/logger'
+
+jest.mock('mongoose', () => ({
+  connection: {
+    // tslint:disable-next-line no-empty
```

Comment:
Are these `tslint:disable` comments required as the value of the next line isn't an empty function?